src/app/grid-toolbar/grid-toolbar-custom.sample.html

```diff
@@ -36,6 +34,14 @@ <h4 class="sample-title">Toolbar</h4>
                 </ng-template>
 
             </igx-grid>
+
+            <br><igx-switch [(ngModel)]="this.igxGrid1.showToolbar">Toolbar</igx-switch>
```

Comment:
Since you have added a template reference variable to the grid (#grid1), you don't need to declare the igxGrid1 and simply use "grid1" in ngModel.
``` html
[(ngModel)]="grid1.columnHiding"
```

Reply:
Addressed.